packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -267,33 +263,44 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
              * At this point the test has finished, we might want to store a list of all the actions at this point and await them all to make sure they finish.
              * Feel free to put anything in the debug object. It's likely to get big. You can also put a range of actions if you're just trying to hone in on
              * what changed.
+             *
+             * This is the validation and data storage part of the test.
              */
-            const debugObject = {
+
+            const stats = {
+                ...actionStats,
                 actionCount: actionsList.length,
-                actionsList,
                 errorCount: errorList.length,
-                errorList,
-                ...actionStats,
-                handleRecord: handleTracker.handleActionRecord,
             };
 
+            fs.mkdirSync(`nyc/testData-${seed}`, { recursive: true });
```

Comment:
Why nyc?  I know that's used for other test output in some cases but does it apply here?  I think that's based on the name of the package that is used or something.  No harm in using the name if it doesn't apply, just curious.

Reply:
It's just so when you run the tests, that it's ignored for git.

Reply:
If this test gets fleshed out more, we should be using something that would report to the nyc folder most likely.